docs/modules/servers/partials/operate/cli.adoc

```diff
@@ -272,6 +288,49 @@ level. Note: syntax is similar to what was exposed previously.
 {cli} GetGlobalMaxStorageQuota
 ....
 
+=== Manage DropList
+
+The DropList, also known as the mail blacklist, is a collection of
+domains and email addresses that are denied from sending emails within
+the system.
+
+Owner scopes:
+
+* `global`: contains entries that are blocked across all domains and
+addresses within the system. Entries in the global owner scope apply
+universally and affect all users and domains.
+* `domain`: each domain can have its own droplist, which contains
+entries specific to that domain.
+* `user`: allow to customize personalized droplist of blocked domains
+and email addresses.
+
+See example commands for `domain` owner scope, `owner@owner.com`
+owner, `evil@devil.com` denied entity.
+
+Add Entry to the DropList
+
+....
+{cli} AddDropListEntry domain owner@owner.com evil@devil.com
```

Comment:
If the example is about the domain, shouldn't the command be then:

```
{cli} AddDropListEntry domain owner.com evil@devil.com
```

Idem for below examples?